### PR TITLE
chore(fixit): clean up vision web samples

### PR DESCRIPTION
**Pull request by balajismaniam**
## Description

Sample clean-up for Vision Web Detection.

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: vision/snippets/web/web_detect_test.py [hunk 1]
```diff
@@ -20,17 +20,16 @@
...
-    assert 'description' in out.lower()
+    assert "description" in out.lower()
 
 
 @pytest.mark.flaky(max_runs=3, min_passes=1)
-def test_detect_web_gsuri(capsys):
-    file_name = ('gs://{}/vision/landmark/pofa.jpg'.format(
-                 ASSET_BUCKET))
+def test_detect_web_gsuri(capsys) -> None:
+    file_name = "gs://{}/vision/landmark/pofa.jpg".format(ASSET_BUCKET)
```
Comment (rsamborski) [inline vision/snippets/web/web_detect_test.py:32]: ditto: Please convert to an f-string as we use them in other places of this sample already.

**Review by rsamborski — CHANGES_REQUESTED** [review]

File: vision/snippets/web/web_detect.py [hunk 2]
```diff
@@ -26,20 +26,29 @@
 import argparse
 
 from google.cloud import vision
+
 # [END vision_web_detection_tutorial_imports]
 
 
 def annotate(path):
-    """Returns web annotations given the path to an image."""
+    """Returns web annotations given the path to an image.
```
Comment (telpirion) [inline vision/snippets/web/web_detect.py:34]: suggestion: while you're in this file, you might consider adding type hints to arguments too.

File: vision/snippets/web/web_detect.py [hunk 3]
```diff
@@ -51,47 +60,65 @@ def annotate(path):
...
 
         for page in annotations.pages_with_matching_images:
-            print(f'Url   : {page.url}')
+            print(f"Url   : {page.url}")
 
     if annotations.full_matching_images:
-        print('\n{} Full Matches found: '.format(
-              len(annotations.full_matching_images)))
+        print(
+            "\n{} Full Matches found: ".format(
```
[the remark this answers is not in the document]
Comment (balajismaniam) [reply vision/snippets/web/web_detect.py]: Updated.

File: vision/snippets/web/web_detect.py [hunk 4]
```diff
@@ -51,47 +60,65 @@ def annotate(path):
...
+    Args:
+        annotations: The web annotations (WebDetection object) from which
+        the features should be parsed and printed.
+    """
     # [START vision_web_detection_tutorial_print_annotations]
     if annotations.pages_with_matching_images:
-        print('\n{} Pages with matching images retrieved'.format(
-            len(annotations.pages_with_matching_images)))
+        print(
+            "\n{} Pages with matching images retrieved".format(
```
[the remark this answers is not in the document]
Comment (balajismaniam) [reply vision/snippets/web/web_detect.py]: Updated.

File: vision/snippets/web/web_detect.py (hunk 2 above)
Comment (balajismaniam) [reply vision/snippets/web/web_detect.py:34]: Added.

File: vision/snippets/web/web_detect.py [hunk 5]
```diff
@@ -51,47 +60,65 @@ def annotate(path):
...
 
         for image in annotations.full_matching_images:
-            print(f'Url  : {image.url}')
+            print(f"Url  : {image.url}")
 
     if annotations.partial_matching_images:
-        print('\n{} Partial Matches found: '.format(
-              len(annotations.partial_matching_images)))
+        print(
+            "\n{} Partial Matches found: ".format(
```
[the remark this answers is not in the document]
Comment (balajismaniam) [reply vision/snippets/web/web_detect.py]: Updated.

File: vision/snippets/web/web_detect_test.py (hunk 1 above)
Comment (balajismaniam) [reply vision/snippets/web/web_detect_test.py:32]: Updated.

File: vision/snippets/web/web_detect.py [hunk 6]
```diff
@@ -51,47 +60,65 @@ def annotate(path):
...
 
         for image in annotations.partial_matching_images:
-            print(f'Url  : {image.url}')
+            print(f"Url  : {image.url}")
 
     if annotations.web_entities:
-        print('\n{} Web entities found: '.format(
-              len(annotations.web_entities)))
+        print(
+            "\n{} Web entities found: ".format(len(annotations.web_entities))
```
[the remark this answers is not in the document]
Comment (balajismaniam) [reply vision/snippets/web/web_detect.py]: Updated.

**Review by rsamborski — APPROVED** [review]

**Review by telpirion — APPROVED** [review]

**Pull request merged**